src/mixins/dom.js

```diff
@@ -13,21 +13,27 @@ export function setDomMixin(mixin) {
 }
 
 export default {
-  getEl,
+  getEl({el, context}) {
+    if (context) {
+      return Backbone.$(el, context);
```

Comment:
This _cannot_ be `$context.find(el)` because it is breaking.  For some reason we have a test that account for `Backbone.$(someEl, notTheParent);` and expects it to work.. which I guess jquery handles?  super weird.

Anyhow.. I think the `parentEl` interface of Regions should go away.... so this can be more optimized soon.